apps/evm/lib/evm/message_call.ex

```diff
@@ -105,10 +106,19 @@ defmodule EVM.MessageCall do
         message_call.current_exec_env
         |> Map.put(:account_interface, n_exec_env.account_interface)
 
+      refund = n_sub_state.refund + message_call.current_sub_state.refund
+
+      sub_state = %SubState{
+        refund: n_sub_state.refund + message_call.current_sub_state.refund,
+        selfdestruct_list:
+          n_sub_state.selfdestruct_list ++ message_call.current_sub_state.selfdestruct_list,
+        logs: message_call.current_sub_state.logs
```

Comment:
We don't have to append logs from the message call like we did with the selfdestruct_list?